example16/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Je to maven project a blbě se nám bude pracovat, jak ho máme zanořený v Learningu. 
Robota jsem dal sice tak samo, ale taky by se mohl vyextrahovat. 

Tak vytvoř nové moduly / maven projecty, ať jsou na stejné úrvoni jak Learning ... to ale taky nepůjde tak snadno, protože Learning není maven project ... udělejme to teda takto: 

Learning bude obsahovat pom, kde budou naspecifikované moduly. robot7/tvoje crud apka/ a pak nase priklady .... kdyz spustim mvn clean install na tom parentovi LEarning, tak se mi zbuildi vsechny zminene moduly(robot/learning/crud) ... jak budu v jednotlivych modulech, tak budu buildit jednotlive moduly. Jak to vyresis, tak pak udelam review na crud. ... to je takovy ukol na maven, to se ti bude hodit